waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/mapping/model/DatabaseMappingImplTest.java

```diff
@@ -682,6 +682,50 @@ public void transformOutboundGetTableResult() throws Exception {
     assertThat(transformedResult.getTable(), is(sameInstance(result.getTable())));
     assertThat(transformedResult.getTable().getDbName(), is(OUT_DB_NAME));
     assertThat(transformedResult.getTable().getTableName(), is(TABLE_NAME));
+    assertFalse(transformedResult.getTable().isSetViewExpandedText());
+    assertFalse(transformedResult.getTable().isSetViewOriginalText());
+  }
+
+  @Test
+  public void transformOutboundGetTableResultWithView() throws Exception {
+    Table table = new Table();
+    table.setDbName(DB_NAME);
+    table.setTableName(TABLE_NAME);
+    table.setViewOriginalText("select cid from " + DB_NAME + "." + "foo");
+    table.setViewExpandedText("select `foo`.`cid` from `" + DB_NAME + "`.`foo`");
+    GetTableResult result = new GetTableResult();
+    result.setTable(table);
+    GetTableResult transformedResult = databaseMapping.transformOutboundGetTableResult(result);
+    assertThat(transformedResult, is(sameInstance(result)));
+    assertThat(transformedResult.getTable(), is(sameInstance(result.getTable())));
+    assertThat(transformedResult.getTable().getDbName(), is(OUT_DB_NAME));
+    assertThat(transformedResult.getTable().getTableName(), is(TABLE_NAME));
+    assertThat(transformedResult.getTable().getViewExpandedText(),
+        is("select `foo`.`cid` from `" + OUT_DB_NAME + "`.`foo`"));
+    assertThat(transformedResult.getTable().getViewOriginalText(), is("select cid from " + OUT_DB_NAME + "." + "foo"));
+  }
+
+  String transformOutboundView(String dbName, String text) {
```

Comment:
you are only using this once, maybe best to just get rid of it doesn't add much, might as well write out the expected string in the test much easier to read what is going on

Reply:
Ah yes this shouldn't be here :)